library/include/hipblas.h

```diff
@@ -439,6 +442,28 @@ typedef enum
     HIPBLAS_SIDE_BOTH = 143
 } hipblasSideMode_t;
 
+#endif // HIPBLAS_SHARED_ENUMS
+
+// Ensure hipBLAS enums have expected values
+#if __cplusplus >= 201103L
+
+static_assert(HIPBLAS_OP_N == 111);
+static_assert(HIPBLAS_OP_T == 112);
+static_assert(HIPBLAS_OP_C == 113);
+
+static_assert(HIPBLAS_FILL_MODE_UPPER == 121);
+static_assert(HIPBLAS_FILL_MODE_LOWER == 122);
+static_assert(HIPBLAS_FILL_MODE_FULL == 123);
+
+static_assert(HIPBLAS_DIAG_NON_UNIT == 131);
+static_assert(HIPBLAS_DIAG_UNIT == 132);
+
+static_assert(HIPBLAS_SIDE_LEFT == 141);
+static_assert(HIPBLAS_SIDE_RIGHT == 142);
+static_assert(HIPBLAS_SIDE_BOTH == 143);
+
+#endif // __cplusplus
```

Comment:
In C++11, static_assert requires two arguments. The single-argument version was added in C++17.